docs/document/content/reference/api-change-history/shardingsphere-jdbc/spring-boot-starter.cn.md

```diff
@@ -49,9 +49,9 @@ spring.shardingsphere.rules.sharding.broadcast-tables[0]= # 广播表规则列
 spring.shardingsphere.rules.sharding.broadcast-tables[1]= # 广播表规则列表
 spring.shardingsphere.rules.sharding.broadcast-tables[x]= # 广播表规则列表
 
-spring.shardingsphere.sharding.default-database-strategy.xxx= # 默认数据库分片策略
-spring.shardingsphere.sharding.default-table-strategy.xxx= # 默认表分片策略
-spring.shardingsphere.sharding.default-key-generate-strategy.xxx= # 默认分布式序列策略
+spring.shardingsphere.rules.sharding.default-database-strategy.xxx= # 默认数据库分片策略
+spring.shardingsphere.rules.sharding.default-table-strategy.xxx= # 默认表分片策略
+spring.shardingsphere.rules.sharding.default-key-generate-strategy.xxx= # 默认分布式序列策略
```

Comment:
This is the change history, is it need to modify?

Reply:
Yes, it's used in '/examples' like [Line 34](https://github.com/apache/shardingsphere/blob/c89217636d579a050d59a95e0e532980d5488dd3/examples/shardingsphere-jdbc-example/single-feature-example/extension-example/custom-sharding-algortihm-example/spi-based-sharding-algorithm-example/spi-based-sharding-spring-boot-jpa-example/src/main/resources/application-sharding-databases-tables.properties)